misk-policy/src/main/kotlin/misk/policy/opa/OpaPolicyEngine.kt

```diff
@@ -45,3 +51,21 @@ inline fun <reified T : OpaRequest, reified R : OpaResponse> OpaPolicyEngine.eva
 ): R {
   return evaluateWithInput(document, input, T::class.java, R::class.java)
 }
+/**
+ * Evaluate / Query a document with given input of raw JSON.
+ * This will connect to OPA via a retrofit interface and perform a /v1/data/{document} POST.
+ * This consumes raw JSON for corner cases where developers need to do queries the automatic
```

Comment:
nit: "...need to do queries **that** the automatic..."

Reply:
Done.